routes/api.php

```diff
@@ -34,6 +34,7 @@
     Route::delete('/donations/{id}', [DonationController::class, 'destroy']);
     //petitions routes  
     Route::get('/petitions', [PetitionController::class, 'index']);
+    Route::get('/petitions/filter', [PetitionController::class, 'filter']);
```

Comment:
This filter stuff should be done at the endpoint `/petitions` and the params should be `time_period`

Reply:
Done